modules/howtos/pages/kv-operations.adoc

```diff
@@ -221,12 +245,18 @@ var result = await collection.Touch("document-key", TimeSpan.FromSeconds(30),
 
 See the https://github.com/couchbase/docs-sdk-c/blob/release/3.1/modules/devguide/examples/c/expiration.cc[code sample^] for use in context.
 
-include::6.5@sdk:shared:partial$documents.adoc[tag=exp-note]
+
 
 
 == Atomic Counters
 
-The value of a document can be increased or decreased atomically using `Binary.Increment()` and `.Binary.Decrement()`.
+The numeric content of a document can be manipulated using https://docs.couchbase.com/sdk-api/couchbase-c-client/group__lcb-counter.html[lcb_RESPCOUNTER^]. Counter opeations treat the document
+as a numeric value (the document must contain a parseable integer as its content). This value may then be incremented
+or decremented.
+[source,c]
+----
+include::example$atomic-counters.cc[tag=atomic-counter]
+----
 ////
```

Comment:
This doc leaves a lot hidden behind ////
We need to go through and check what can be uncommented